### ISSUE TITLE
mapbox setTerrain undefined

### ISSUE THREAD

**Issue by crnismrk** [keplergl/kepler.gl#1541]
**Describe the bug**
I am writing kepler.gl app that depends on mapbox 3D terrain feature. 
Until lates update I was able to get mapbox ref, call getMap() to gain access to map object. That map object had setTerrain function used to set terrain source and exaggeration. 
After update setTerrain is undefined. 

**To Reproduce**
1. Embed kepler.gl in react app as described here https://docs.kepler.gl/docs/api-reference/get-started 
2. Pass a getMapboxRef callback to a Kepler component, callback receives mapbox_reference.
3. In a callback function call mapbox_reference.getMap, it returns map_object
4. Call map_object.setTerrain. It will result in console error "Uncaught TypeError: map.setTerrain is not a function". Inspect map object, setTerain function is not defined

**Expected behavior**
Map object setTerrain should not be undefined. 

**Desktop (please complete the following information):**
 - OS: Windows10
 - Browser: Chrome 91.0., Firefox 89.0
 - Version: kepler.gl@2.5.2

**Comment by heshan0131** [comment]
This doesn't seem to be a kepler action. Can you confirm which version of mapboxgl support `setTerrain`?

**Comment by crnismrk** [comment]
Ok, I just took some time to investigate it. **It is not a bug**, my fault, I forgot that I was experimenting with yarn resolutions and changed mapbox-gl dependency to a version that supports setTerrain. Later new kepler version came out and I deleted all changes, installed dependencies and was surprised that setTerrain is gone (undefined).

Are there any plans for kepler to change react-map-gl dependency to a version v6.0.0 and above?

I really appreciate all your work here, It is amazing library. Sorry for this mistake.
Best regards,
